android/app/build.gradle

```diff
@@ -35,13 +35,14 @@ android {
         targetCompatibility JavaVersion.VERSION_1_8
     }
 
+    aaptOptions { cruncherEnabled = false }
```

Comment:
This line was added to stop finished-with-error-gradle-task-assemblerelease-failed-with-exit-code-1